README.md

```diff
@@ -62,6 +66,8 @@ You will find the [Book source code](http://pragprog.com/titles/tpantlr2/source_
 
 ## Additional grammars
 
-[This repository](https://github.com/antlr/grammars-v5) is a collection of grammars verified with ANTLR 5 where the
-root directory name is the all-lowercase name of the language parsed
+As of now, there is no collection of grammars for ANTLR 5, but we plan
+to grow such collection in [grammars-v5](https://github.com/antlr/grammars-v5), which is currently empty.
+
+Until we get grammars for ANTLR 5, you can take a look at [this repository](https://github.com/antlr/grammars-v4); it is a collection of grammars for ANTLR 4 where the root directory name is the all-lowercase name of the language parsed
```

Comment:
how about `grammars verified for ANTLR 4` ?

Reply:
changed